src/gui/attributetable/qgsattributetablemodel.cpp

```diff
@@ -294,6 +301,9 @@ void QgsAttributeTableModel::fieldFormatterRemoved( QgsFieldFormatter *fieldForm
 
 void QgsAttributeTableModel::attributeValueChanged( QgsFeatureId fid, int idx, const QVariant &value )
 {
+  // Skip all updates if an edit command is running
+  if ( mBulkEditCommandRunning )
+    return;
   QgsDebugMsgLevel( QStringLiteral( "(%4) fid: %1, idx: %2, value: %3" ).arg( fid ).arg( idx ).arg( value.toString() ).arg( mFeatureRequest.filterType() ), 3 );
 
   for ( SortCache &cache : mSortCaches )
```

Comment:
Is the sorting also updated properly after a bulk update?

Reply:
and ... shouldn't that be qgis::as_const anyway?

Reply:
No: it's not

Reply:
What should I call to update the sorting?

Reply:
> and ... shouldn't that be qgis::as_const anyway?

I didn't look at that line, I trust the author ;)

```
2544391a9 src/gui/attributetable/qgsattributetablemodel.cpp (Matthias Kuhn      2018-05-04 15:34:34 +0200)     for ( SortCache &cache
```